src/core/server/app/authenticators/oauth2/state.ts

```diff
@@ -0,0 +1,57 @@
+import cookie from "cookie";
+import crypto from "crypto";
+import { CookieOptions, Request, Response } from "express";
+import compare from "tsscmp";
+
+const COOKIE_NAME = "oauth2:state";
+const COOKIE_OPTIONS: CookieOptions = {
+  httpOnly: true,
+  secure: true,
```

Comment:
Should support local development without https.

Reply:
☝️